v0/pipeline/stages/custom/stage.yaml

```diff
@@ -21,10 +21,10 @@ properties:
 #      "$ref": "#/definitions/FailureStrategyConfig"
   identifier:
     type: string
-    pattern: "^[a-zA-Z_][0-9a-zA-Z_]{0,63}$"
+    pattern: "^[a-zA-Z_][0-9a-zA-Z_]{0,127}$"
```

Comment:
changes here are not required. You can delete this file

Reply:
done